_includes/specials/taproot/en/11-vaults-with-taproot.md

```diff
@@ -0,0 +1,63 @@
+and future key aggregation proposals made possible by [BIP340][]. However, these schemes come with new
+safety challenges. Aiming to be used for cold storage, vault protocols are more conservative in
+their choices and would probably be the last to use these new technologies, if it uses them at all.
+
+By switching to taproot, vaults would also benefit from a slight privacy and efficiency improvement
+due to the use of merkle branches and shorter BIP340 signatures (especially for multi-party ones).
+For instance the *unvault* output script in a multi-party setup with 6 "cold" keys and 3 "active" keys
+(with a threshold of 2) could be represented as a Taproot of depth 2 with leaves:
+
+- `<X> CSV <active key 1> CHECKSIGADD <active key 2> CHECKSIGADD 2 EQUAL`
+- `<X> CSV <active key 2> CHECKSIGADD <active key 3> CHECKSIGADD 2 EQUAL`
+- `<X> CSV <active key 3> CHECKSIGADD <active key 1> CHECKSIGADD 2 EQUAL`
+- `<cold key 1> CHECKSIGADD <cold key 2> CHECKSIGADD <cold key 3> CHECKSIGADD <cold key 4> CHECKSIGADD <cold key 5> CHECKSIGADD <cold key 6> CHECKSIGADD 6 EQUAL`
+
+This is about 76.5 vbytes for the cheapest (happy) spending path and about 176.0 vbytes for the costliest (dispute) one.
+Compared to the roughly 134.75 vbytes (happy) and 189.5 vbytes (dispute) by using the following P2WSH:
+
+```text
+IF
+  6 <cold key 1> <cold key 2> <cold key 3> <cold key 4> <cold key 5> <cold key 6> 6 CHECKMULTISIG
+ELSE
+  <X> CSV DROP
+  2 <active key 1> <active key 2> <active key 3> 3 CHECKMULTISIG
+ENDIF
+```
+
+It is to be noted that although the revocation branch can be hidden in case of a successful
+spending (and if using a multisig threshold, its existence and the number of participants
+obfuscated), the privacy gains are minimal as the vault usage pattern would be trivially
+identifiable onchain.
+
+Finally, vault protocols, like most pre-signed transactions protocols, would largely benefit from
+further proposed Bitcoin upgrades based on taproot such as [BIP118][]'s [SIGHASH_ANYPREVOUT][topic sighash_anyprevout]. Although requiring further caution
+and protocol tweaks `ANYPREVOUT` and `ANYPREVOUTANYSCRIPT` would enable for instance rebindable *cancel*
+signatures, which could largely reduce interactivity and allow 0(1) signature storage. This is
+particularly interesting for the *emergency* signature in the Revault protocol, as it would largely
+reduce the DoS attack surface. `ANYPREVOUTANYSCRIPT` signatures would also enable some forms of
+covenants that could eventually be made more flexible by the introduction of new sighash types.
+
+[^0]:
+    If known in advance you could pre-sign the *spend* transaction with a specific nSequence, but then
+    you don't need an alternative spending path with "active" keys at all. Also, you don't usually know
+    how you are going to spend your coins by the time you receive them.
+
+[darosior]: https://github.com/darosior
+[revault]: https://revault.dev/
```

Comment:
Just noticed this, i think a link to the Github would be more appropriate :)
https://github.com/revault